VOKMockUrlProtocol.m

```diff
@@ -30,6 +30,9 @@
 static NSString *const HTTPHeaderContentTypeFormUrlencoded = @"application/x-www-form-urlencoded";
 static NSString *const HTTPHeaderContentTypeJson = @"application/json";
 
+//255 HFS+ filename limit - 5 for file extension suffix
+static NSInteger const MaxBaseFilenameLength = 255 - 5;
```

Comment:
I don't think the 5 is guaranteed in the future... but it's safe enough for now.  Do you think there might be a system constant for the 255?